Documentation/teaching/labs/device_drivers.rst

```diff
@@ -397,7 +397,7 @@ calling the macros / functions below:
 All macros / functions return 0 in case of success and another value in case of
 error and have the following roles:
 
-   * ``put_user`` put in the user-space at the address address value of the val;
+   * ``put_user`` put in the user-space at the address a value of the val;
      Type can be one on 8, 16, 32, 64 bit (the maximum supported type depends on the
```

Comment:
That's not really a mistake because the name of the address parameter is 'address'. But I think it is more clear with your rephrashing so I'm ok with the change.

Reply:
Ok, I will update so.

Reply:
Just updated according to @tavip 's comment and moved this change to the second commit, as this is just 'wordsmithing' rather than 'fix', as @dbaluta explained.